modify java.md, python.md, and configure-introduction.md (#260)
tikv/website: content/docs/5.1/develop/clients/java.md

        weight: 1
---

> This document guides you through how to use [Java Client](https://github.com/tikv/client-java) in TiKV.

Reviewer: ```suggestion
This document guides you on how to use [Java Client](https://github.com/tikv/client-java) in TiKV.
```

}
```

> These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduce network overhead and increase performance under certain workloads.

Reviewer: ```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduces network overhead and increases performance under certain workloads.
```

Author: Here I think `which` means `batch variants` and it is in plural form.

Reviewer: Then please remove the comma before `which`.

Author: Done. PTAL~

> These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduce network overhead and increase performance under certain workloads.

Author: ```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`) which considerably reduce network overhead and increase performance under certain workloads.
```

> These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduce network overhead and increase performance under certain workloads.

Author: ```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`) which considerably reduce network overhead and increase performance under certain workloads.
```

> You can find all the functions `RawKVClient` supports [here](https://github.com/tikv/client-java/blob/master/src/main/java/org/tikv/raw/RawKVClient.java).

Reviewer: ```suggestion
You can find all functions supported by `RawKVClient` [in this JAVA file](https://github.com/tikv/client-java/blob/master/src/main/java/org/tikv/raw/RawKVClient.java).
```
